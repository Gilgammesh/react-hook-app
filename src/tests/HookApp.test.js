import React from "react";
import { shallow } from "enzyme";
import "@testing-library/jest-dom";
import HookApp from "../HookApp";

describe("Pruebas unitarias en el componente HookApp.js", () => {
  /************************************************************************************/
  test("se debe mostrar el componente <HookApp /> correctamente", () => {
    const wrapper = shallow(<HookApp />);
    expect(wrapper).toMatchSnapshot();
  });
});
