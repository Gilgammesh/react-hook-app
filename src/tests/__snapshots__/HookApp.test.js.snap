// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas unitarias en el componente HookApp.js se debe mostrar el componente <HookApp /> correctamente 1`] = `
<div>
  Hola Mundo
</div>
`;
